src/CoreWCF.Primitives/src/CoreWCF/Dispatcher/ServiceDispatcher.cs

```diff
@@ -61,7 +63,7 @@ public async Task<IServiceChannelDispatcher> CreateServiceChannelDispatcherAsync
              this, /*wasChannelThrottled*/ false, sessionIdleManager);
 
             var channelDispatcher = channelHandler.GetDispatcher();
-            channel.ChannelDispatcher = channelDispatcher;
+         //   channel.ChannelDispatcher = channelDispatcher;
```

Comment:
Why's this line commented out?

Reply:
this is set in RequestDelegate once the task is completed... we discussed this.. let me know if you think otherwise..